main.go

```diff
@@ -3,17 +3,16 @@ package main
 import (
 	"fmt"
 	"net/http"
+	"sync"
 )
 
-var validUserIDs = []string{
-	"user1",
-	"user2",
-	"user3",
-}
+var validUserIDs sync.Map
```

Comment:
:+1:

Reply:
Good idea using the `sync.Map`. I might or might not ask questions about why you picked it ;).